package.json

```diff
@@ -4,14 +4,11 @@
     "Thaddee Tyl <thaddee.tyl@gmail.com>",
     "Jan Keromnes <janx@linux.com>"
   ],
-  "name": "camp",
-  "description": "Streamed Web Server with easy Ajax, EventSource, Templating, Static pages; Middleware approach and extensibility.",
-  "version": "17.2.2",
-  "homepage": "http://espadrine.github.com/sc/",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/espadrine/sc.git"
-  },
+  "name": "@shields_io/camp",
+  "description": "Fork of Scoutcamp maintained for Shields",
+  "version": "18.0.0",
+  "homepage": "https://github.com/badges/sc",
+  "repository": "badges/sc",
```

Comment:
The format of this parameter has changed. Any reason why? The previous type/url elements did seem consistent with the [official documentation](https://docs.npmjs.com/creating-a-package-json-file).

Reply:
Ah, good question. `repository` supports a shorthand for GitHub repos: https://docs.npmjs.com/files/package.json#repository.

It's in the spec and well-supported by tools.

Reply:
Yes, I didn't read that bit fully, thanks!